src/material/icon/icon.scss

```diff
@@ -43,6 +39,12 @@ mat-icon {
   &.mat-ligature-font[fontIcon]::before {
     content: attr(fontIcon);
   }
+
+  // Makes button icon overflow visible to fix b/250063405
+  // Undoes previous fix to Issue #11826 in preference of
+  // fixing to improve accessibility by not having the icon
+  // get cut off
+  overflow: visible;
```

Comment:
Updated based on recommendation.

Reply:
> I expect this to be quite breaking. Couldn't we resolve the issue by setting something like this?
> 
> ```
> .mat-mdc-button-base .mat-icon {
>   flex-shrink: 0;
> }
> ```

Hi @crisbeto, I know you already approved this, but as I was going back and re-testing the approved changes on the buttons, the icons are still getting cut off on the bottom as you can see from [this screencast](https://screencast.googleplex.com/cast/NTgwNzEzMTg5MzY5NDQ2NHwyMjM0ZjhhOC00Mg). The flex-shrink: 0 only fixes the icon from being cropped/cutoff on the left/right, but not the bottom.

Let me know your thoughts.

Reply:
I updated the latest change to add `height: min-content` to `.mat-mdc-button-base .mat-icon` to prevent the bottom of the icon from being cut off in buttons. Here are screenshots showing that it works with the updated height for with and without text-spacing applied:

[No text-spacing screenshot](https://screenshot.googleplex.com/5FKM5SK2trwbwue)
[With text-spacing screenshot](https://screenshot.googleplex.com/AiVowXwA828LQ9a)